deploy/helm/kubecf/values.yaml

```diff
@@ -181,6 +181,16 @@ features:
         password: ~
         username: ~
 
+# Enable or disable instance groups for the different test suites.
+# Only smoke tests should be run in production environments.
+testing:
+  acceptance_tests:
```

Comment:
Since there are many `acceptance-tests` for CF components ([check here](https://github.com/cloudfoundry?utf8=%E2%9C%93&q=acceptance-tests&type=&language=)), I think it makes sense for us to start being explicit here as `cf-acceptance-tests` as we will definitely have other acceptance-tests added. Could you also use kebab-case instead of snake-case?

Reply:
An example of accessing the Helm value as kebab-case: https://github.com/SUSE/kubecf/blob/04bca19d055fdbc522564a6bcfd27a794c0527c4/deploy/helm/kubecf/assets/operations/instance_groups/tcp-router.yaml#L1

Reply:
@f0rmiga Yes, I'm aware of the kebab-case issues and how to work around them. I wonder if it's worth it, though.

In the end I opted for snake-case because it makes things easier and the current `values.yaml` is inconsistent anyway. For example, `system_domain` and `kube.storage_class` and others are snake_case and it's even used for instance groups already, e.g. in `sizing.tcp_router`.

I think it would be good to streamline it one way or the other, ideally before the release. wdyt?

Reply:
The project uses snake-case for pretty much everything but respects names. I'm aware that we merged code that skipped that informal rule even after I requested changes related to that in sizing. I'd like to have that fixed before release - to any direction we decide to follow.

I can see that from a developer perspective making everything snake-case is easier - we just use dots everywhere.

An open question: for the final user - the cluster operator, is it easier to use the real name of the release, instance-group, job, etc, or just make anything snake-case everywhere?

Reply:
Note that from a developer perspective, we can make the code way simpler in many cases by using the real names, being that kebab or snake-case. It can be achieved by iterating over the keys of a map to generate template snippets.

Reply:
My personal opinion is that using snake-case everywhere is the best compromise because it's consistent on the user-facing side **and** simplifies the code consuming the values. But then again that's just me, it should probably be decided in a larger group.

Reply:
Let's go with snake case - seems to be easier to deal with in code, and it's consistent with other BOSH things like properties.

Reply:
I'll leave it in snake case for now and created https://github.com/SUSE/kubecf/issues/258 as a follow-up issue.